src/scipp/coords/transform_coords.py

```diff
@@ -47,6 +48,8 @@ def transform_coords(x: Union[DataArray, Dataset],
                               Default is True.
     :param keep_inputs: Keep consumed input coordinates or attributes.
                         Default is True.
+    :param quiet: If True, no log output is produced. Otherwise, ``transform_coords``
+                  produces a log of its actions.
```

Comment:
Mention the default?

Reply:
Should be shown in the signature int he docs. See also this discussion: https://github.com/scipp/ess/pull/93#discussion_r794379578

There are certainly cases were it makes sense to repeat the default value. But I wouldn't do it too often. Is this case a case where we should mention it?

Reply:
Sphinx sure, but what about `help(myfunc)`?

Reply:
Shown but not great: (example for a version w/o `quiet`)
```python
>>> help(sc.transform_coords)
Help on function transform_coords in module scipp.coords.transform_coords:

transform_coords(x: Union[scipp._scipp.core.DataArray, scipp._scipp.core.Dataset], targets: Union[str, Iterable[str]], graph: Dict[Union[str, Tuple[str, ...]], Union[str, Callable]], *, rename_dims: bool = True, keep_aliases: bool = True, keep_intermediate: bool = True, keep_inputs: bool = True) -> Union[scipp._scipp.core.DataArray, scipp._scipp.core.Dataset]
    Compute new coords based on transformations of input coords.
    
...
```

The question mark in Jupyter is much better:
```python
[1]: sc.transform_coords?
Signature:
sc.transform_coords(
    x: Union[scipp._scipp.core.DataArray, scipp._scipp.core.Dataset],
    targets: Union[str, Iterable[str]],
    graph: Dict[Union[str, Tuple[str, ...]], Union[str, Callable]],
    *,
    rename_dims: bool = True,
    keep_aliases: bool = True,
    keep_intermediate: bool = True,
    keep_inputs: bool = True,
) -> Union[scipp._scipp.core.DataArray, scipp._scipp.core.Dataset]
Docstring:
Compute new coords based on transformations of input coords.

...
```

The problem is also that if we write the default value in the docstring, we have to keep it up to date. So I would be against doing it in all cases.

Reply:
Ok.